### ISSUE TITLE
Form validation feedback doesn't show

### ISSUE THREAD

**Issue by rhaamo** [greyli/bootstrap-flask#15]
I'm using a form auto generated with the bootstrap/form.html macros like that:
```
{{ bf4.render_form(forgot_password_form, action=url_for_security('forgot_password'), render_kw={'name': "reset_password_form"}, horizontal_columns=('lg', 3, 5), form_type='horizontal', button_map={'submit': 'success'}) }}
```

But when the server returns an error, the form does get the feedback:
```
<form action="/reset" method="post" class="form" role="form" name="reset_password_form">
        <input id="csrf_token" name="csrf_token" type="hidden" value="xxx">
    <div class="form-group row required">
                <label class="form-control-label  col-lg-3" for="email">Adresse email</label>
                <div class=" col-lg-5">
                    <input class="form-control is-invalid" id="email" name="email" required="" type="text" value="xxx" style="background-repeat: no-repeat; background-attachment: scroll; background-size: 16px 16px; background-position: calc(100% - 4px) 50%; background-image: url(&quot;moz-extension://e62d5f8c-1e48-4df9-b88c-35970aeec8c2/skin/icon-light.png&quot;); cursor: auto;">
                </div>
        <div class="offset-lg-3 col-lg-5">
                            <div class="invalid-feedback">Cet utilisateur n'existe pas</div>
    </div>
        </div>
            <div class="form-group row">
        <div class="offset-lg-3 col-lg-5">
            <input class="btn btn-success" id="submit" name="submit" type="submit" value="R?cup?rer le mot de passe" data-_extension-text-contrast="">
    </div>
    </div>    
    </form>
```

```<div class="invalid-feedback">Cet utilisateur n'existe pas</div>```

But it does not shows on the page because ```.invalid-feedback``` is ```display: none;```. Browsing loooong issues on bootstrap4 tracker the only thing I see is like adding ```d-block``` to this div to force it showing up.

Any ideas for this ?

**Comment by rhaamo** [comment]
I was using bootstrap 4.1.3 and Bootstrap-Flask 1.0.8.

**Comment by greyli** [comment]
I can't reproduce this issue. With the version you provided, I created a minimal application, but the error message was displayed normally. I also noticed that the `.invalid-feedback` class was set to `display: block` eventually (see the snapshot below).

![snipaste_2018-11-13_20-33-45](https://user-images.githubusercontent.com/12967000/48413789-bf3d5580-e783-11e8-9381-2798417c3bb3.png)

**Comment by rhaamo** [comment]
I made more tests and discovered that if I remove ```form_type='horizontal'``` from my ```render_form()``` call, the ```.invalid-feedback``` thing works fine.

But by using the default (basic) I loose control over the horizontal columns sizes.

**Comment by greyli** [comment]
Fixed in [1.0.9](https://github.com/greyli/bootstrap-flask/releases/tag/1.0.9), please upgrade the package and give it a try:
```
$ pip install -U bootstrap-flask
```

**Comment by greyli** [comment]
I’ll close this issue for now. Please feel free to comment further or re-open this issue if you wish.
